src/pages/index.vue

```diff
@@ -1457,10 +1457,15 @@ export default {
     },
   },
   mounted() {
-    window.addEventListener('resize', this.handleResize);
+    this.resizeListener = window.addEventListener('resize', this.handleResize);
```

Comment:
Should define `resizeListener` in `data`?

Reply:
I think no need to put it in data for reactivity

Reply:
Best to define it somewhere clearly as a class atteibute though

Reply:
Defined in `$options`